src/main/webapp/app/overview/course-competencies/course-competencies-details.component.html

```diff
@@ -78,26 +78,24 @@ <h3 class="mb-0">
                 <div class="w-50 mx-auto my-2">
                     <jhi-competency-rings
                         [progress]="progress"
-                        [confidence]="confidence"
                         [mastery]="mastery"
                         [hideProgress]="judgementOfLearningEnabled && promptForJolRating && judgementOfLearning === undefined"
                     />
                 </div>
                 @if (!promptForJolRating || !judgementOfLearningEnabled || judgementOfLearning !== undefined) {
                     <jhi-side-panel class="d-block mt-3" [panelHeader]="'artemisApp.competency.table.panelHeader' | artemisTranslate">
                         <div class="row mb-1 g-0">
-                            <div class="col-7">{{ 'artemisApp.competency.progress' | artemisTranslate }} <jhi-help-icon text="artemisApp.competency.progressDescription" /></div>
-                            <div class="col-5">{{ progress }} %</div>
-                        </div>
-                        <div class="row mb-1 g-0">
-                            <div class="col-7">
-                                {{ 'artemisApp.competency.confidence' | artemisTranslate }} <jhi-help-icon text="artemisApp.competency.confidenceDescription" />
+                            <div class="col-7">{{ 'artemisApp.competency.mastery' | artemisTranslate }} <jhi-help-icon text="artemisApp.competency.masteryDescription" /></div>
```

Comment:
I think this should be implemented with `jhiTranslate` instead of `| artemisTranslate` here (client guidelines)

```suggestion
                            <div class="col-7"><span jhiTranslate="artemisApp.competency.mastery"></span><jhi-help-icon text="artemisApp.competency.masteryDescription" /></div>
```